apps/photos/src/components/PhotoViewer/FileInfo/MapButton.tsx

```diff
@@ -0,0 +1,9 @@
+import { Button, ButtonProps, styled } from '@mui/material';
+import { CSSProperties } from '@mui/material/styles/createTypography';
+
+export const MapButton = styled((props: ButtonProps) => (
```

Comment:
we are not providing `navigation` no ?
its just a text no ?
we can make it a div for now ?